### DISCUSSION TITLE
Build a list of Trust Models

### DISCUSSION THREAD

**Discussion by a-fox** [trustoverip/tswg-trust-registry-tf#65] [General]
As per the AP in #50, we should gather a list of different trust models to understand the base mechanics and requirements. This should help us to understand better how different use cases view trust registries / trust decisions. 

[The initial list will be built in a Google slide](https://docs.google.com/presentation/d/1E5-MB-je1ybuFYnxPlMdqAyTELBzfE9Z7IEvovFx-SA/edit#slide=id.g1cf6fdbdf0b_0_0), before moving it later to Markdown/GDoc, once there has been some coherence.

Link to Google Slide deck: https://docs.google.com/presentation/d/1E5-MB-je1ybuFYnxPlMdqAyTELBzfE9Z7IEvovFx-SA/edit#slide=id.g1cf6fdbdf0b_0_0

**Feel free to add a slide of your favorite trust model!**

**Comment by andorsk** [comment]
Note: Discussion slides from Tim on slide 12. Presented on Jan 19, ToIP TRTF NA meeting. https://docs.google.com/presentation/d/1E5-MB-je1ybuFYnxPlMdqAyTELBzfE9Z7IEvovFx-SA/edit#slide=id.g1cf6fdbdf0b_0_0

related to https://github.com/trustoverip/tswg-trust-registry-tf/discussions/62
